src/node/api_clients/rest.py

```diff
@@ -1,15 +1,31 @@
+import requests
+from src.env_vars import API_REQUEST_TIMEOUT
 from src.libs.custom_logger import get_custom_logger
 import json
 from urllib.parse import quote
-from src.node.api_clients.base_client import BaseClient
 
 logger = get_custom_logger(__name__)
 
 
-class REST(BaseClient):
+class REST:
```

Comment:
Do you think it's worth removing the `BaseClient`?

Reply:
Yes, because we only have one protocol now (REST), so don't think it makes sense to keep an abstract class just for that

Reply:
Well, you can consider two possible uses for that:
1. The possibility that a new client type comes into existence in the future, although that might not happen, and according to this point alone I'd agree with you.
2. But more important, in my opinion: Inheritance (or any other code deduplication mechanism, such as functions) does not only help with code duplication, but also with legibility and understandability. By abstracting code (into a function, class, etc.) you're labeling that under a concept (in this case `BaseClient`), so you don't need to read all lines to understanding, just that one word evokes all the correct meaning.

In any case, I'm fine if you decide to go down this path. I just figured it wouldn't hurt pointing out this other possibility.

Reply:
Good points, I agree now that it's a plus to have this BaseClient. Reverted that change. 
Thanks!